deepchem/utils/openvino_model.py

```diff
@@ -71,26 +116,59 @@ def __init__(self,
     self._batch_size = batch_size
     self._outputs: List = []
 
+    self.input_size: List = []
+    self.last_input_shapes: List = []
+    self.net: IENetwork = None
+    self.input_names: List = []
+
   def _read_torch_model(self, generator: Iterable[Tuple[Any, Any, Any]]):
     """
     Prepare PyTorch model for OpenVINO inference:
       1. Convert into ONNX format
       2. Read network from memory
     NOTE: We do not load model in __init__ method because of training.
     """
-    import torch
+    # import dgl
+    import torch.nn.functional as F
 
     # We need to serialize ONNX model with real input shape.
     # So we create a copy of the generator to take first input.
     generator_copy, generator = itertools.tee(generator, 2)
+
+    # list with array of objects deepchem.feat.graph_data.GraphData (len 4)
     inputs, _, _ = next(generator_copy)
-    assert (len(inputs) == 1), 'Not implemented'
-    inp_shape = list(inputs[0].shape)
-    inp_shape[0] = self._batch_size
+
+    if (type(self._torch_model).__name__ == 'CGCNNModel'):
+      # Create Graph object
+      inputs, _, _ = self._torch_model._prepare_batch((inputs, None, None))
+      
+      # Data for message_func cat
+      u, v, _ = inputs.edges(form='all')
```

Comment:
Will `u, v = inputs.edges()` will be the same?

Reply:
No
```
ValueError: too many values to unpack (expected 2)
```

Reply:
Maybe `u, v = inputs.edges(form='uv')`?